tests/acquisition/covidcast/test_covidcast_meta_cache_updater.py

```diff
@@ -8,11 +8,11 @@
 # third party
 import pandas
 
-from delphi.epidata.acquisition.covidcast.covidcast_meta_cache_updater import get_argument_parser, \
```

Comment:
should we move this file from `tests/acquisition/covidcast/` to `tests/maintenance/` to match the `src` file that got moved?